dlib/cuda/cudnn_dlibapi.cpp

```diff
@@ -131,11 +131,11 @@ namespace dlib
             cudnn_activation_descriptor(
                 cudnnActivationMode_t mode,
                 cudnnNanPropagation_t reluNanOpt,
-                double reluCeiling
+                double coef
```

Comment:
I've renamed this to `coef`, because that's the name used in the [CUDNN API](https://docs.nvidia.com/deeplearning/cudnn/api/index.html#cudnnSetActivationDescriptor) and it can be used for other purposes other than relu ceiling